pkg/apps/index.jsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from "cockpit";
+import $ from "jquery";
```

Comment:
I thought we weren't supposed to introduce new jQuery code? From what I can see you mostly use it for element lookups like `$("#id")`, which can be replaced with `document.getElementById("id")`, and signal handlers like `$(obj).on()`, which can be replaced with `obj.addEventListener()`. The only other use that I see is `$(obj).triggerHandler()`; I haven't used that myself yet, but it looks like [.dispatchEvent()](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/dispatchEvent) could replace this?

Reply:
Upps, I need to work on this.